lib/ufe/StagesSubject.cpp

```diff
@@ -25,12 +25,32 @@
 #include <ufe/scene.h>
 #include <ufe/sceneNotification.h>
 #include <ufe/transform3d.h>
+#include <ufe/attributes.h>
```

Comment:
This should go at the top of this group, though `path.h` is unfortunately in the wrong spot too.

Reply:
Done.